llm4pddl/utils.py

```diff
@@ -356,3 +358,58 @@ def str_to_identifier(x: str) -> str:
         https://stackoverflow.com/questions/5297448
     """
     return hashlib.md5(x.encode('utf-8')).hexdigest()
+
+
```

Comment:
can we actually move this function into a new file in the `scripts/` directory? we probably won't use it while running the regular pipeline. also, this will save us from requiring test coverage.

Reply:
also, make sure that we have an easy way to run this code! if it's a script, then we should be able to just run the script (most ideally, using command line arguments to specify the input and output paths)